LICENSE-3rdparty.yml

```diff
@@ -777,10 +967,16 @@ third_party_libraries:
   - package_name: webpki
     package_version: 0.22.0
-    license: License specified in file (/tmp/dd-cargo/registry/src/github.com-1ecc6299db9ec823/webpki-0.22.0/LICENSE)
+    license: License specified in file (/Users/paul.legranddescloizeaux/.cargo/registry/src/github.com-1ecc6299db9ec823/webpki-0.22.0/LICENSE)
     licenses:
-      - license: License specified in file (/tmp/dd-cargo/registry/src/github.com-1ecc6299db9ec823/webpki-0.22.0/LICENSE)
+      - license: License specified in file (/Users/paul.legranddescloizeaux/.cargo/registry/src/github.com-1ecc6299db9ec823/webpki-0.22.0/LICENSE)
```

Comment:
Oh I messed up here when recreating the license file

Reply:
no worries, BTW I was looking for some other ways to handle the licenses. And so far haven't found any good/better alternative